src/kube-runtime/src/init

```diff
@@ -66,7 +66,7 @@ function exit_handler()
       ;;
     *)
       echo "Unknown exit code, platform error"
-      exit 248
+      exit 1
```

Comment:
Just exit the orignal exit code, it is already comparable

Reply:
We don't check if the exitCode is duplicated with some user predefined code here. Exit 1 is safety.

Reply:
You already checked all predefined code by:
  pattern:
    runtimeContainerPatterns:
    - exitCode: XXX
So it is safe to return, otherwise, you may have inconsistent with our spec

Reply:
It is same safe as if your bash process crashed/killed before exit 1. So exit 1 is unnecessary and overwrite real raw comparable exit code